src/App.tsx

```diff
@@ -1,28 +1,63 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useMemo, useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Errors } from './types/Errors';
+import { ErrorNotify } from './components/ErrorNotify/ErrorNotify';
+import { Filter } from './components/Filter/Filter';
+import { FilterType } from './types/FilterType';
+import { filterTodos } from './utils/helpers';
+
+const USER_ID = 12041;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<Errors | null>(null);
+  const [filterType, setFilterType] = useState(FilterType.All);
+
+  const onErrorNotifyClose = () => {
+    setError(null);
+  };
+
+  const onFilterSelect = (filterT: FilterType) => {
+    setFilterType(filterT);
+  };
+
+  useEffect(() => {
+    const renderTodos = async () => {
+      try {
+        const todosFromServer = await getTodos(USER_ID);
+
+        setTodos(todosFromServer);
+      } catch (e) {
+        setError('Unable to load todos');
+        setTimeout(() => {
+          setError(null);
+        }, 3000);
```

Comment:
- You will do it for all errors, so it's better to move this logic outside.
- Also, you have `onErrorNotifyClose` function, you can use it